src/OrchardCore/OrchardCore.Data.YesSql/Options/YesSqlOptionsConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Microsoft.Extensions.Options;
+using OrchardCore.Data.YesSql;
+using YesSql.Services;
+
+namespace OrchardCore.Data
```

Comment:
Why not `OC.Data.YesSql`?

Reply:
Yes I agree, it's just a beginning will see tomorrow ;)

For now just because it is OC.Data in the other files under the same folder.

Reply:
I have the same comment for other files in `OC.Data.YesSql` namespace. The PR looks good but we need to fix the unit test

Great job as usual